extensions/blocks/donations/donations.php

```diff
@@ -26,6 +26,63 @@ function register_block() {
 		array(
 			'render_callback' => __NAMESPACE__ . '\render_block',
 			'plan_check'      => true,
+			'attributes'      => array(
```

Comment:
Block attributes need to be defined server-side because default values of attributes defined client-side are not available in `render_callback`.

See https://github.com/WordPress/gutenberg/issues/7342 or https://github.com/WordPress/gutenberg/pull/22491